doc/source/docs/api/geoseries.rst

```diff
@@ -0,0 +1,167 @@
+=========
+GeoSeries
+=========
+.. currentmodule:: geopandas
+
+Constructor
+-----------
+.. autosummary::
+   :toctree: api/
+
+   GeoSeries
+
+General methods and attributes
+------------------------------
+
+.. autosummary::
+   :toctree: api/
+
+   GeoSeries.area
+   GeoSeries.bounds
+   GeoSeries.total_bounds
+   GeoSeries.length
+   GeoSeries.geom_type
+   GeoSeries.distance
+   GeoSeries.representative_point
+   GeoSeries.exterior
+   GeoSeries.interiors
+   GeoSeries.x
+   GeoSeries.y
+
+Unary predicates
+----------------
+
+.. autosummary::
+   :toctree: api/
+
+   GeoSeries.is_empty
+   GeoSeries.is_ring
+   GeoSeries.is_simple
+   GeoSeries.is_valid
+
+
+Binary Predicates
+-----------------
+
+.. autosummary::
+   :toctree: api/
+
+   GeoSeries.geom_almost_equals
+   GeoSeries.contains
+   GeoSeries.crosses
+   GeoSeries.disjoint
+   GeoSeries.geom_equals
+   GeoSeries.intersects
+   GeoSeries.overlaps
+   GeoSeries.touches
+   GeoSeries.within
+   GeoSeries.covers
+   GeoSeries.covered_by
+
+
+Set-theoretic Methods
+---------------------
+
+.. autosummary::
+   :toctree: api/
+
+   GeoSeries.difference
+   GeoSeries.intersection
+   GeoSeries.symmetric_difference
+   GeoSeries.union
+
+Constructive Methods
```

Comment:
Nitpick: the below items are not all "methods", but some are also "attributes", when using the strict python terminology. Not sure this is a problem though, or if we should separate them in two lists below or so, ..